src/components/MdSteppers/MdStep.vue

```diff
@@ -42,6 +42,16 @@
         }
       }
     },
+    computed: {
+      isActive () {
+        return this.id === this.MdSteppers.activeStep
+      },
+      tabIndex () {
+        return !this.isActive
+          ? -1
+          : false
```

Comment:
Should `false` be `0` instead? Pretty sure that should be a number.

Reply:
False is because it should not be applied to element.

Reply:
Just following the HTML specs. Also for confirmation, made a quick Google search: https://stackoverflow.com/a/47322194

Reply:
Yes but binded tabindex is not a same as html attribute.

Reply:
@xxRockOnxx you can check this with devtool
https://codepen.io/VdustR/pen/EpNGzy

Pass `null`, `false`, `undefined` to attribute on native element node will unset it.

Reply:
@VdustR I see. Is that because of Vue or not? Because if not I think it's up to the browser what to do with it and that might cause inconsistencies.

Reply:
It's wrote in the documentation.

https://vuejs.org/v2/guide/syntax.html#Attributes

```
If isButtonDisabled has the value of null, undefined, or false, the disabled attribute will not even be included in the rendered <button> element.
```